Marlin/UltiLCD2.cpp

```diff
@@ -260,19 +260,20 @@ void lcd_menu_main()
 
 
 #define BREAKOUT_PADDLE_WIDTH 21
-//Use the lcd_cache memory to store breakout data, so we do not waste memory.
-#define ball_x (*(int16_t*)&lcd_cache[3*5])
-#define ball_y (*(int16_t*)&lcd_cache[3*5+2])
-#define ball_dx (*(int16_t*)&lcd_cache[3*5+4])
-#define ball_dy (*(int16_t*)&lcd_cache[3*5+6])
+//Use the lcd_cache memory to store breakout data, so we do not waste memory. (i.e. 23 bytes of RAM saved, wow.)
+#define breakout_buf    ((char*)&lcd_cache)
+#define ball_x          (*(int16_t*)&breakout_buf[3*5])
```

Comment:
Why the 3*5? If the lcd_cache is used for anything else then it would make more sense to use remove the 3+5?

Reply:
The first 15 bytes are used to store the 3 rows of 5 'bricks' data but didn't get their own variable name. Code is ugly, perhaps on purpose to hide this easter egg a bit?